src/components/GameSwitchboard.js

```diff
@@ -1,8 +1,35 @@
+//GameSwitchboard.js made by Jason Iqbal for SweatEquity, a division of sweatyPits.
+
 import React from 'react';
 import House from './House.js';
 import Menu  from './Menu.js';
+import { 
+  Stitch, 
+  RemoteMongoClient,
+  AnonymousCredential } from 'mongodb-stitch-browser-sdk';
 
 class GameSwitchboard extends React.Component {
+    constructor(props) {
+        super(props);
+        
+        this.client = Stitch.initializeDefaultAppClient('sweatequity-cbpxb');
+        this.db = this.client.getServiceClient(RemoteMongoClient.factory, 'mongodb-atlas').db('highScores');
+        
+    }
+  
+    submitScoreInfo = (props) => {
+        this.client.auth.loginWithCredential(new AnonymousCredential()).then( user => {
+            let scoreCol = this.db.collection('sweatEquity');
+            scoreCol.insertOne({
+              name  : document.getElementById('scoreName').value,
```

Comment:
We shouldn't have to access the element by id here. The submit handler should have a reference to the element where the user types already.